addons/web_editor/static/lib/odoo-editor/src/commands/toggleList.js

```diff
@@ -26,12 +26,9 @@ HTMLElement.prototype.oToggleList = function (offset, mode = 'UL') {
         restoreCursor();
     } else {
         const list = insertListAfter(this, mode, [this]);
```

Comment:
Commit title is too long:
![image](https://github.com/odoo/odoo/assets/24205914/aa3ceeac-255b-4234-bb51-33301d94f007)

Reply:
There's a typo now. How about `[FIX] web_editor: preserve only dir attribute on toggle list`?

```diff
-  [FIX] web_editor: toggleList to preserve ony dir attribute
+ [FIX] web_editor: preserve only dir attribute on toggle list
```

Your changes do make it clearer, thank you. I guess I can agree with this and if we notice later more attributes should be moved to the list then we can do that.

Also:
```diff
- Commit [1] implemented an approach to maintain all the attributes when
+ Commit [1] implemented an approach to preserve all the attributes when
```
```diff
- converting to a list. Which is beneficial for `p` element where paragraph
+ converting to a list. This is beneficial for `p` elements since they will no
```
```diff
- element will no longer be in dom. However, elements other that `p`, will always
+longer be in the DOM. However, elements other that `p` will always
```
(then redo line-breaking accordingly)